mocks/mockData.js

```diff
@@ -67,125 +67,141 @@ const artists = [
+    role: 'PLANNER',
   },
   {
-    personId: "d2ab07f3-2865-47d8-a3f9-60c5db744a6d",
-    organisationId: "c1c2d01f-83d9-4ef8-a0f4-2f53dcd41c41",
-    foreName: "ShiftCount",
-    familyName: "User",
-    grade: "AA",
-    role: "PLANNER",
+    personId: 'd2ab07f3-2865-47d8-a3f9-60c5db744a6d',
+    organisationId: 'c1c2d01f-83d9-4ef8-a0f4-2f53dcd41c41',
+    foreName: 'ShiftCount',
+    familyName: 'User',
+    grade: 'AA',
+    role: 'PLANNER',
   },
   {
-    personId: "e8eccdc9-4f80-4848-875d-983f28f5ab79",
-    organisationId: "0de94b00-805a-4bc2-97f6-b9ac9025c4c0",
-    foreName: "Aja",
-    familyName: "Wunsch",
-    grade: "HEO",
-    role: "PLANNER",
+    personId: 'e8eccdc9-4f80-4848-875d-983f28f5ab79',
+    organisationId: '0de94b00-805a-4bc2-97f6-b9ac9025c4c0',
+    foreName: 'Aja',
+    familyName: 'Wunsch',
+    grade: 'HEO',
+    role: 'PLANNER',
   },
   {
-    personId: "bd1e2d45-4ca1-41b3-b7cd-f988ccf05853",
-    organisationId: "79590ae1-ae1c-4b83-9a97-9ccce7a97d68",
-    foreName: "Fidel",
-    familyName: "Kirlin",
-    grade: "SEO",
-    role: "PLANNER",
+    personId: 'bd1e2d45-4ca1-41b3-b7cd-f988ccf05853',
+    organisationId: '79590ae1-ae1c-4b83-9a97-9ccce7a97d68',
+    foreName: 'Fidel',
+    familyName: 'Kirlin',
+    grade: 'SEO',
+    role: 'PLANNER',
   },
   {
-    personId: "c5f63d63-a714-4349-b833-88c20386ed3b",
-    organisationId: "0de94b00-805a-4bc2-97f6-b9ac9025c4c0",
-    foreName: "Toi",
-    familyName: "Reynolds",
-    grade: "G6",
-    role: "USER",
+    personId: 'c5f63d63-a714-4349-b833-88c20386ed3b',
+    organisationId: '0de94b00-805a-4bc2-97f6-b9ac9025c4c0',
+    foreName: 'Toi',
+    familyName: 'Reynolds',
+    grade: 'G6',
+    role: 'USER',
   },
   {
-    personId: "1356abea-2e41-4410-8283-801d510ca3c7",
-    organisationId: "0de94b00-805a-4bc2-97f6-b9ac9025c4c0",
-    foreName: "Juan",
-    familyName: "Bauch",
-    grade: "AO",
-    role: "MANAGER",
+    personId: '1356abea-2e41-4410-8283-801d510ca3c7',
+    organisationId: '0de94b00-805a-4bc2-97f6-b9ac9025c4c0',
+    foreName: 'Juan',
+    familyName: 'Bauch',
+    grade: 'AO',
+    role: 'MANAGER',
   },
   {
-    personId: "18f36830-d3af-4927-a6f0-fd2c3219a48b",
-    organisationId: "0de94b00-805a-4bc2-97f6-b9ac9025c4c0",
-    foreName: "Alise",
-    familyName: "Adams",
-    grade: "HEO",
-    role: "ADMIN",
+    personId: '18f36830-d3af-4927-a6f0-fd2c3219a48b',
+    organisationId: '0de94b00-805a-4bc2-97f6-b9ac9025c4c0',
+    foreName: 'Alise',
+    familyName: 'Adams',
+    grade: 'HEO',
+    role: 'ADMIN',
   },
 ];
 
 const newTimeCardEntry = {
-  startTime: "09:00",
-  finishTime: "15:00",
-  startDate: "2022-02-24",
-  finishDate: "2022-02-24",
+  meta: {
+    next: null,
+  },
+  items: [
+    {
+      id: 'c0a80040-82cf-1986-8182-cfedbbd50003',
+      tenantId: '00000000-0000-0000-0000-000000000000',
+      version: 0,
+      ownerId: 1,
+      timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
+      shiftType: 'string',
```

Comment:
shiftType: 'string', Maybe it should be a null or empty string rather than string 'string'?